_layouts/api.html

```diff
@@ -9,6 +9,12 @@
         {% if page.pill == 'overview' %} class="active" {% endif %}>
         <a href="/documentation/api">Overview</a></li>
         <li
+        {% if page.pill == 'overview' %} class="active" {% endif %}>
+        <a href="/documentation/api#android">Android</a></li>
+        <li
+        {% if page.pill == 'overview' %} class="active" {% endif %}>
+        <a href="/documentation/api#spring">Android</a></li>
+        <li
```

Comment:
I think the API page is larger than the others, and it covers more variants.

NB: I just spotted a typo in my patch: the link to Spring  is labeled "Android"